notebooks/00_quick_start/run_notebook_on_aml.ipynb

```diff
@@ -226,7 +220,48 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "run.wait_for_completion(show_output=True)"
+    "run = exp.submit(cfg)\n",
+    "run"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Above cell should output similar table as below\n",
+    "![Experiment submit output](https://recodatasets.blob.core.windows.net/images/aml_papermill_cell_output.jpg)\n",
+    "After clicking \"Link to Azure Portal\", experiment run details tab looks like this\n",
+    "![Azure Portal Experiment](https://recodatasets.blob.core.windows.net/images/aml_papermill_portal.jpg)\n",
+    "Experiment output is out.ipynb. You can download and view this file locally.\n",
+    "![Output file](https://recodatasets.blob.core.windows.net/images/aml_papermill_outputs_tab.jpg)\n",
+    "#### Jupyter widget\n",
+    "\n",
+    "Jupyter widget can watch the progress of the run.  Like the run submission, the widget is asynchronous and provides live updates every 10-15 seconds until the job completes."
```

Comment:
"You can use a Jupyter widget to monitor the progress of the run. Like the run submission, the widget is asynchronous and provides live updates every 10-15 seconds until the job completes."

Reply:
Added above line